bin/reporttests.jl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env julia
+#!julia
```

Comment:
Why?

Reply:
Using `#!julia` fails with the tests I've added. Specifically:
```
IOError: could not spawn `/Users/cvogt/.julia/dev/TestReports/bin/reporttests.jl`: no such file or directory (ENOENT)
```
where as `#!/usr/bin/env julia` does not.

Reply:
the FAQ recommends `/usr/bin/env julia`
https://docs.julialang.org/en/v1/manual/faq/#How-do-I-pass-options-to-julia-using-#!/usr/bin/env?

Reply:
I'll leave this as it is since we don't need to pass in arguments into the `#!` at this time.